design/move-gh-org.md

```diff
@@ -0,0 +1,89 @@
+# Plan for moving the Velero GitHub repo into the VMware GitHub organization
+
+Currently, the Velero repository sits under the Heptio GitHub organization. With the acquisition of Heptio by VMware, it is due time that this repo moves to one of the VMware GitHub organizations. This document outlines a plan to move this repo to the VMware Tanzu (https://github.com/vmware-tanzu) organization.
+
+## Goals
+
+- List all steps necessary to have this repo fully functional under the new org
+
+## Non Goals
+
+- Highlight any step necessary around setting up the new organization and its members
+
+## Action items
+
+### Todo list
+
+#### Pre move
+
+- [ ] PR: Blog post communicating the move. https://github.com/heptio/velero/issues/1841. Who: TBD.
+- [ ] PR: Find/replace in all Go, script, yaml, documentation, and website files: `github.com/heptio/velero -> github.com/vmware-tanzu/velero`. Who: a Velero developer; TBD
+- [ ] PR: Update website with the correct GH links. Who: a Velero developer; TBD
+- [ ] PR: Change deployment and grpc-push scripts with the new location path. Who: a Velero developer; TBD
+
+#### Move
+
+- [ ] Use GH UI to transfer the repository to the VMW org; must be accepted within a day. Who: new org owner; TBD
+- [ ] Make owners of this repo owners of repo in the new org. Who: new org owner; TBD
+- [ ] Update Travis CI. Who: Any of the new repo owners; TBD
+- [ ] Add DCO for signoff check. (https://probot.github.io/apps/dco/). Who: Any of the new repo owners; TBD
+- [ ] Each individual developer should point their origin to the new location: `git remote set-url origin git@github.com:vmware-tanzu/velero.git`
```

Comment:
As I understand https://help.github.com/en/articles/transferring-a-repository, the old remote will work, so this can be done by devs separately, it shouldn't block the overall move.